stories/FileTree.stories.tsx

```diff
@@ -1,72 +1,85 @@
 /* eslint @typescript-eslint/camelcase: 0 */
 import React from 'react';
+import { Store } from 'redux';
 import { storiesOf } from '@storybook/react';
 
-import FileTree from '../src/components/FileTree';
+import configureStore from '../src/configureStore';
+import FileTree, {
+  PublicProps as FileTreeProps,
+} from '../src/components/FileTree';
 import { createInternalVersion } from '../src/reducers/versions';
 import { fakeVersion, fakeVersionEntry } from '../src/test-helpers';
+import { renderWithStoreAndRouter } from './utils';
 
-const version = createInternalVersion({
```

Comment:
That seems unnecessary to me. Storybook outputs static, compiled files for prod and, in dev, there is hot reload with webpack.

Reply:
Well, it would still slow down the startup of all stories but, of course, it's most likely YAGNI. It's just a bad habit for us to get into. No worries, I can change it back 😛

Reply:
In the current code, the `version` creation code executed at [startup time](https://github.com/mozilla/addons-code-manager/blob/fc93cfa4b1592ba8f2cddff3c5a1f0532bfdc479/stories/setup/config.tsx#L27). In my patch, it only executed in [the handler function](https://github.com/mozilla/addons-code-manager/blob/fc93cfa4b1592ba8f2cddff3c5a1f0532bfdc479/stories/FileTree.stories.tsx#L67). We should always avoid executing code during startup because this pollutes the startup process. In any case, the optimization probably won't matter and since everyone was confused about it, I reverted my change.